miasm2/core/utils.py

```diff
@@ -83,31 +85,46 @@ def __setitem__(self, asked_key, value):
 
             # Bound can only be reached on a new element
             if (self._size >= self._max_size):
-                most_commons = [key for key, _ in self._counter.most_common()]
+                most_common = sorted(self._counter.iteritems(),
+                                     key=itemgetter(1), reverse=True)
```

Comment:
Actually, this is the implementation used in `collections.Counter`, I took it as is, trusting the python guys.
But this is a good question. :)
It seems to have better performance than the lambda function, even in trivial cases: http://stackoverflow.com/questions/17243620/operator-itemgetter-or-lambda.

For the import part, it is imported by `collections` anyway, and this module imports `collections`.